tests/git/test_git_commands.py

```diff
@@ -542,35 +544,35 @@ def test_api_integration_and_secrets_get_unique_names(
     [
         (
             "@repo/branches/main/",
-            "@repo/branches/main/",
+            "@repo/branches/main",
             ["@repo/branches/main/s1.sql", "@repo/branches/main/a/S3.sql"],
         ),
         (
             "@repo/branches/main/a",
-            "@repo/branches/main/",
+            "@repo/branches/main",
```

Comment:
To align the behavior. The `StagePath` accepts both inputs (with or without trailing /) but the representation is always without / suffix